apps/CricMyBrainAndroidApp/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
```

Comment:
We can add .idea complete folder to .gitignore.
It is IntelliJ specific and gets created everytime project is imported.